ui/Screen/Wallet.svelte

```diff
@@ -130,7 +159,7 @@
       </div>
     {:else}
       <ConnectWallet
-        onConnect={wallet.connect}
+        onConnect={connectWallet}
         connecting={$wallet.status === Status.Connecting} />
```

Comment:
Could we now just show the QR code directly on this page, like @brandonhaslegs [wanted](https://github.com/radicle-dev/radicle-upstream/issues/1964)?

Reply:
I requires a bit more effort but in effect we should be able to do this now.

Reply:
Cool, we can follow-up on that later.